src/content/docs/Company/ceo.md

```diff
@@ -6,15 +6,13 @@ sidebar:
   order: 99
 ---
 
-Modeled after [GitLab's CEO doc](https://about.gitlab.com/handbook/ceo/).
-
 The page is about Eric, CEO of Shorebird. The page is intended to be helpful. If
 it sounds pretentious or overbearing, please let me know so I can fix it.
 Thanks.
 
 ## CEO bio
 
-Eric is the founder and CEO of Shorebird. Shorebird makes code push for Flutter.
+Eric is the founder and CEO of Shorebird. Shorebird is the Flutter company.
```

Comment:
Don't know if you need the "flutter company line here" this page is about you, not the company. If anything should we be putting "The Flutter Company" as our tagline instead of "Helping businesses succeed with Flutter."

Reply:
The original intent of this blurb was to be the quick copy/paste for when a short bio was needed.  happy to refactor it.